src/VertexManager.sol

```diff
@@ -409,7 +409,9 @@ contract VertexManager is Initializable, UUPSUpgradeable, OwnableUpgradeable, Re
     /// @notice Returns the price on Vertex of a given by product.
     /// @param id The ID of the product to get the price of.
     function getPrice(uint32 id) public view returns (uint256) {
-        return IClearinghouse(endpoint.clearinghouse()).getOraclePriceX18(id);
+        // If id is 0 (quote), return default price.
+        if (id == 0) return 1e18;
```

Comment:
Am I reading it correctly that the oracle price used to revert on an invalid ID? Should we make our wrapper maintain that behavior? What's the utility for returning a price for an invalid product?

Reply:
updated on #49